### DISCUSSION TITLE
Copy new repo failed

### DISCUSSION THREAD

**Discussion by rebullm** [greenelab/lab-website-template#187] [General] [answered]
Hi! In trying to upgrade from earlier version to v1.1, I used "use this template" and got the following error: 
![image](https://user-images.githubusercontent.com/93389344/234085365-64e2b868-92b8-43c0-bbf7-ce80ba4285b5.png)
<html>
<body>

build-site / build-siteGet Pages site failed
--
build-site / build-siteAxiosError: Request failed with status code 404

Any idea how to move forward? 
Thanks, 
Meg

**Comment by vincerubinetti** [comment, marked as answer]
Hi Meg,

That error is due to the build workflow not being able to find the URL of your GitHub Pages site, which happens when you don't have GitHub Pages enabled and/or a `gh-pages` branch. To build the site successfully, the automated workflows need to be able to get the site URL. Did you [follow these steps](https://greene-lab.gitbook.io/lab-website-template-docs/getting-started/set-up-your-site), particularly steps 2 and 3?

I'm assuming this is the repo you're referring to:
https://github.com/rebullm/Bennett-Lab-website-v1.1/

`first-time-setup` doesn't seem to have been run yet. Another thing that's strange to me is that the "on push to main" has been triggered/run two times, but there has only been [one commit to main](https://github.com/rebullm/Bennett-Lab-website-v1.1/commit/daea1dc4461e04f91e0f15f139be8fa827ef70a5).

<img width="278" alt="Screenshot 2023-04-24 at 3 27 36 PM" src="https://user-images.githubusercontent.com/8326331/234096318-20839f59-a88c-4d1a-8841-3b8a50298392.png">

You'll see that the first run of it has a gray slash icon, because it was skipped. That is intentional. You'll see the reason [in the workflow file in a comment](https://github.com/rebullm/Bennett-Lab-website-v1.1/actions/runs/4789805090/workflow#L18): on the first run, nothing is setup yet, so the workflow would fail if run.

Did you manually re-run that workflow or something like that? From what I can tell, you didn't, in which case I can't explain why GitHub Actions would trigger the workflow twice for a single commit. If you didn't manually re-run it, it is safe to ignore, because it is expected (not a bug), and you can just follow the instructions like normal.

**Reply by rebullm** [reply]
Hi Vince, I hadn't gotten to steps 2 and 3, because I got the error on step 1. I'll keep going and ignore it. 
Thanks, 
Meg
